### DISCUSSION TITLE
Why is v0.12.5 tag/branch missing from the repo?

### DISCUSSION THREAD

**Discussion by vadimkantorov** [facebook/lexical#5392] [General]
I've used the latest release https://github.com/facebook/lexical/releases/tag/v0.12.5 and it worked for my usecase. Now I'd like to fork the repo and checkout tag v0.12.5, but it seems deleted now.
<img width="960" alt="image" src="https://github.com/facebook/lexical/assets/1041752/197853d6-7806-4bd5-9daa-47a009a633d8">

Thank you :)

**Comment by evgost** [comment]
The [branch](https://github.com/facebook/lexical/pull/5343) got merged into main and deleted afterwards.

**Reply by vadimkantorov** [reply]
Thanks for clarification. I hope it was by accident, or maybe even the deleted tag could be restored? It's quite nasty to have a release and no way to chceckout that commit/branch

**Comment by etrepum** [comment]
The tag isn't deleted, that is GitHub's warning message that lets you know that the tag has a commit that doesn't belong to any branch. This is what I've been told the current process is for creating a release

* New version is tagged and branched at the same time by a workflow
* Branch PR is created
* Branch PR reviewed, approved, and then *squash merged* to main (this makes it a different commit, leaving the tag as an 'orphan' commit, but the contents are otherwise the same)
* Version is published to npm by a workflow (technically using the commit from main but its contents should be indistinguishable from the tag)
* Release is created in github with manual release notes (presumably from the tag)
